website/src/pages/demo.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../components/Layout'
+import { useHubSpot } from '../hooks/hubSpot'
+
+const Demo: FunctionComponent<PageProps> = props => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: 'baf7d112-bb19-4b95-851e-a83e7b214b9b',
+        targetId: 'form',
+        chiliPiper: true,
+    })
+
+    return (
+        <Layout
+            location={props.location}
+            meta={{
+                title: 'Request a Demo | Sourcegraph',
+                description: `From developer onboarding to incident response, see how companies of all sizes use Sourcegraph to solve the industry's most challenging code problems.`,
+            }}
+            heroAndHeaderClassName="bg-gradient-green-blue"
+            hero={
+                <div className="container pt-6 pb-4">
+                    <h1 className="font-weight-bolder">Request a demo</h1>
```

Comment:
Figma has semi-bold for this looks like

Reply:
Actually, I see that there are two; was looking at this
![Screen Shot 2022-03-29 at 9 56 12 AM](https://user-images.githubusercontent.com/43454804/160627840-1da2183b-1a5c-460e-b815-3bf6dfe6e414.png)

Maybe this was updated?

Reply:
I believe the Figma artboard at the bottom is the correct artboard to follow which also meets feature parity with the current https://info.sourcegraph.com/demo-request page. Both use a 700 font weight.

Also, Bootstrap doesn't have a semibold font weight utility class so if we want to switch we'll have to kick off a typography sass partial.

Reply:
Our `font-weight-bold` does 600: https://github.com/sourcegraph/about/blob/4f2dba1f1057960b24ebecc78b0f1d9da81f9f43/website/src/css/styles.scss#L20-L22
![Screen Shot 2022-03-29 at 1 55 07 PM](https://user-images.githubusercontent.com/43454804/160675242-201e8635-62d2-4827-9741-e343ed292874.png)

Reply:
> Our `font-weight-bold` does 600:

[oh no lol](https://i.imgflip.com/6am5xb.jpg)

Reply:
lololololol